### ISSUE TITLE
Dual SIM - Separate Notifications?

### ISSUE THREAD

**Issue by macnero** [klinker-apps/messenger-issues#425]
**Feature Request:** Can you add a way to have a separate Notification Sounds for each SIM on a Dual SIM phone?

**Device with issue:** LG V20 H990DS
**Primary device:** LG V20 on Android 7"
**App version:** 2.5.0.1446

**Comment by klinker24** [comment]
Can you tell me the use-case for this? I am not really sure I understand why you would ever want that, over a per-conversation notification sound.

I see it as you are getting a notification for a conversation, not really for an individual message, so it makes sense to me that it wouldn't matter what SIM those are coming from, they are still being saved to the same conversation and coming from the same contact.

So I am a hesitant "no" for whether this will get implemented because it doesn't seem useful for most people, right? But if you can tell me what you see in this feature and convince me of your use case, I would be happy to investigate the possibilities of this further. I just need to know whether or not it makes sense

**Comment by macnero** [comment]
Sorry for not being clear.  Typically those of us with Dual SIM phones have one for Personal and one for Business.  It would be extremely helpful if the messages that come into each line has a unique tone, rather than having to assign a tone to each conversation.  The stock messaging app has this option.

![screenshot_2017-08-22-15-43-13](https://user-images.githubusercontent.com/31256242/29584186-e83dfdf2-8750-11e7-84d7-5eea6b3cecb5.png)


Furthermore, (I may be stretching it a bit) it would be great to have the ability to MUTE one line or the other.

Let me know if I have answered your question.

**Comment by klinker24** [comment]
Well, I see it is available in the native app, but what makes that helpful compared to what Pulse already provides? I can see a few cases here:

1. you get a text from someone that only ever sends messages to your work number. This contact only has your work number, so you could change their conversation ringtone to know when this work contact sends you a text. *No changes are required for this case.*
2. you get a text from someone that only ever sends messages to your personal number. This contact only has (or uses) your personal number, so you can change their conversation ringtone to know when this contact sends you a text. *No changes are required for this case either*
3. you have a contact that sends messages to both your work and your personal numbers, and you want to know if the text coming in is something work related, or personal related. This seems like an extremely unlikely case, am I right? If someone has s conversation going with you, why would they change which phone number they are text you on, based on the message context?

The third one is the only case here that would require any changes for this request, but it just seems like that wouldn't happen. Is that the case you are looking to solve?

I feel as though all of Pulse's per-conversation settings already do a pretty great job of allowing for this functionality

**Comment by macnero** [comment]
I get hundreds of messages per day from both sides unfortunately. To assign a unique tone to every person that sends me a message to me will take forever.  I guess I got used to hearing different tones with the stock messaging app and it is tough to now have the same sound for each line.

Also, on my particular device, I am not getting the SIM "Stamp" that I read about on incoming messages, so it is impossible for me at this point to determine what message is coming from where.  

![screenshot_2017-08-22-15-59-58](https://user-images.githubusercontent.com/31256242/29584965-70868ab0-8753-11e7-856f-85f6629ff778.png)

**Comment by klinker24** [comment]
Tap any message to view the SIM stamp, it is right next to the timestamp.

**Comment by Oublieuse** [comment]
I agree with macnero about this feature. I have it on my very simple dual sim nokia phone (not a smartphone, a good old normal phone), and i can't understand why it is so complicated to have real separated features for dual sims on smartphones. Having two different ringtones is a basic in my opinion. When you use separate sims for work and personal, you need it... this way, you can choose if you have to check your calls/sms or not. For example, you're at home, with your children, having dinner. Your receive a new sms, and your phone is not right next to you. If the ringtone is "personal", you will have a look at it, if for work, you will check it later.
